wagtailpolls/templates/wagtailpolls/poll_list.html

```diff
@@ -25,7 +25,7 @@
 					<td class='created_on'>
 						{% if poll.date_created %}
 						<div class="human-readable-date" title="{{ poll.date_created|date:"d M Y H:i" }}">
-						{{ poll.date_created|timesince }} ago</div>{% endif %}
+						{{ poll.date_created|timesince }} {% blocktrans %}ago{% endblocktrans %}</div>{% endif %}
```

Comment:
For situations like this, where there is something like `<quantity> <modifier>`, it is best to translate the whole phrase, like:

```html+django
{% blocktrans with when=poll.date_created|timesince %}{{ when }} ago{% endblocktrans %}
```

Not all languages follow the same structure as English in where they put modifiers for some types of measurements.